client/my-sites/domains/domain-management/add-google-apps/add-email-addresses-card.jsx

```diff
@@ -128,20 +133,56 @@ const AddEmailAddressesCard = createReactClass( {
 		);
 	},
 
-	allEmailAddressFieldsets() {
+	renderFieldsets() {
+		return this.state.fieldsets.map( ( _, index ) => {
+			return (
+				<Fragment key={ index }>
+					{ index > 0 && <hr /> }
+					<div className="add-email-addresses-card__email-address-fieldsets">
+						{ this.emailAddressFieldset( index ) }
+					</div>
+					<div className="add-email-addresses-card__name-fieldsets">
+						{ this.nameFieldset( index ) }
+					</div>
+				</Fragment>
+			);
+		} );
+	},
+
+	nameFieldset( index ) {
```

Comment:
Not mandatory at all, but we could use the `render` prefix when naming this type of methods.

Reply:
done